### DISCUSSION TITLE
Invitation to Sponsor CyberCup 3.0 Hackathon

### DISCUSSION THREAD

**Discussion by Yash-1903** [community/community#68019] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, 
I would like to extend an invitation to GitHub and it's various associates to be a sponsor of a prestigious event **"CyberCup 3.0"** which is organized by Amity University, Uttar Pradesh. It is a 24hr Hackathon which is scheduled to take place on 13th-14th October, 2023.
The last edition of CyberCup- CyberCup 2.0 saw a great participation of around 280+ participants. There are 3 themes for the Hackathon: 
1) IoT and Healthcare
2) Blockchain and Cybersecurity
3) AI and Cloud Computing.

For further details, Please mail me on the following email id: [redacted due to PII]

Regards,
Yash Gupta

**Comment by itsdotnet** [comment]
Is it in India. 
If is it can i join online?

**Reply by Yash-1903** [reply]
Hi! Thank you for showing your interest, Yes, it is in India and to participate in it you have to be present physically in Amity University Campus.

**Comment by queenofcorgis** [comment]
Hi @Yash-1903 ,

I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by anjuan** [comment]
Hi @Yash-1903, this is the community discussion forum for [GitHub Sponsors](https://github.com/sponsors) which is designed to invest in open source. I think you're trying to request a sponsorship for "CyberCup 3.0". Your hackathon sounds awesome, but this isn't the right place to find what you're looking for. 🙂
